topbeat/main.go

```diff
@@ -1,7 +1,7 @@
 package main
 
 import (
-	topbeat "github.com/elastic/beats/topbeat/beat"
+	topbeat "github.com/dr-toboggan/beats/topbeat/beat"
```

Comment:
That is not a good idea as it would mean our topbeat is built based on your repo :-) Please change it back to elastic.